DOL.WHD.Section14c.DataAccess/Migrations/Configuration.cs

```diff
@@ -1,204 +1,204 @@
+//            context.AddRoleFeature(Roles.CertificationTeamManager, ApplicationClaimTypes.CreateAccount);
+//            context.AddRoleFeature(Roles.CertificationTeamManager, ApplicationClaimTypes.GetRoles);
+//            context.AddRoleFeature(Roles.CertificationTeamManager, ApplicationClaimTypes.ViewAdminUI);
+//            context.AddRoleFeature(Roles.CertificationTeamManager, ApplicationClaimTypes.ViewAllApplications);
+//            context.AddRoleFeature(Roles.CertificationTeamManager, ApplicationClaimTypes.ChangeApplicationStatus);
+
+//            // CertificationTeamMember
+//            context.AddRoleFeature(Roles.CertificationTeamMember, ApplicationClaimTypes.SubmitApplication);
+//            context.AddRoleFeature(Roles.CertificationTeamMember, ApplicationClaimTypes.ViewAdminUI);
+//            context.AddRoleFeature(Roles.CertificationTeamMember, ApplicationClaimTypes.ViewAllApplications);
+//            context.AddRoleFeature(Roles.CertificationTeamMember, ApplicationClaimTypes.ChangeApplicationStatus);
+
+//            // PolicyTeamMember
+//            context.AddRoleFeature(Roles.PolicyTeamMember, ApplicationClaimTypes.SubmitApplication);
+//            context.AddRoleFeature(Roles.PolicyTeamMember, ApplicationClaimTypes.GetAccounts);
+//            context.AddRoleFeature(Roles.PolicyTeamMember, ApplicationClaimTypes.CreateAccount);
+//            context.AddRoleFeature(Roles.PolicyTeamMember, ApplicationClaimTypes.GetRoles);
+//            context.AddRoleFeature(Roles.PolicyTeamMember, ApplicationClaimTypes.ViewAdminUI);
+//            context.AddRoleFeature(Roles.PolicyTeamMember, ApplicationClaimTypes.ViewAllApplications);
+
+//            // SystemAdministrator
+//            context.AddRoleFeature(Roles.SystemAdministrator, ApplicationClaimTypes.GetAccounts);
+//            context.AddRoleFeature(Roles.SystemAdministrator, ApplicationClaimTypes.CreateAccount);
+//            context.AddRoleFeature(Roles.SystemAdministrator, ApplicationClaimTypes.ModifyAccount);
+//            context.AddRoleFeature(Roles.SystemAdministrator, ApplicationClaimTypes.GetRoles);
+//            context.AddRoleFeature(Roles.SystemAdministrator, ApplicationClaimTypes.ViewAdminUI);
+//            context.AddRoleFeature(Roles.SystemAdministrator, ApplicationClaimTypes.ViewAllApplications);
+//            context.AddRoleFeature(Roles.SystemAdministrator, ApplicationClaimTypes.ChangeApplicationStatus);
+
+//            // WageAndHourFieldManager
+//            context.AddRoleFeature(Roles.WageAndHourFieldManager, ApplicationClaimTypes.ViewAdminUI);
+//            context.AddRoleFeature(Roles.WageAndHourFieldManager, ApplicationClaimTypes.ViewAllApplications);
+
+//            // WageAndHourInvestigator
+//            context.AddRoleFeature(Roles.WageAndHourInvestigator, ApplicationClaimTypes.ViewAdminUI);
+//            context.AddRoleFeature(Roles.WageAndHourInvestigator, ApplicationClaimTypes.ViewAllApplications);
+//        }
+
+//    }
+//}
```

Comment:
Can this file be removed since it's all commented out?

Reply:
@mgwalker This code was commented out to fix production deployment issue. I believe @mmurthydol would like to keep this code for historical reason. Madhu, please free feel to correct me.